core/src/main/java/io/grpc/ManagedChannelBuilder.java

```diff
@@ -42,7 +40,8 @@
  * @param <T> The concrete type of this builder.
  */
 public abstract class ManagedChannelBuilder<T extends ManagedChannelBuilder<T>> {
-  public static ManagedChannelBuilder forAddress(String name, int port) {
+  public static <T extends ManagedChannelBuilder<T>> ManagedChannelBuilder<T> forAddress(
```

Comment:
The more I think of this the more it seems `ManagedChannelBuilder<?>` is most precise and proper from a generics standpoint (assuming my testing is correct that it is chainable).